docs/ecto.md

```diff
@@ -0,0 +1,447 @@
+# Data Validation and Persistence with Ecto
+
+Most web applications today need some form of data validation and persistence. In the Elixir ecosystem, we have Ecto to enable this. Before we jump into building database-backed web features, we're going to focus on the finer details of Ecto to give a solid base to build our web features on top of. Let's get started!
+
+Ecto currently has adapters for the following databases:
+
+* PostgreSQL
+* MySQL
+* MSSQL
+* SQLite3
+* MongoDB
+
+Newly generated Phoenix projects include Ecto with the PostgreSQL adapter by default.
+
+For a thorough, general guide for Ecto, check out the [Ecto getting started guide](https://hexdocs.pm/ecto/getting-started.html). For an overview of all Ecto specific mix tasks for Phoenix, see the [mix tasks guide](mix_tasks.html#ecto-specific-mix-tasks).
+
+This guide assumes that we have generated our new application with Ecto integration and that we will be using PostgreSQL. For instructions on switching to MySQL, please see the [Using MySQL Guide](using_mysql.html).
+
+The default Postgres configuration has a superuser account with username 'postgres' and the password 'postgres'. If you take a look at the file ```config/dev.exs```, you'll see that Phoenix works off this assumption. If you don't have this account already setup on your machine, you can connect to your postgres instance by typing ```psql``` and then entering the following commands:
```

Comment:
This paragraph uses "you" instead of "we".

> The default Postgres configuration has a superuser account with username 'postgres' and the password 'postgres'. The configuration file ```config/dev.exs``` works off this assumption. If this account is not already setup on your machine, we can connect to the postgres instance by typing ```psql``` then enter the following commands: